### PR TITLE
First Aid Kits in First Aid Kit Lockers

### PR DESCRIPTION
**Pull request by OolongCow**
White first aid wall lockers now consistently contain a basic first aid kit rather than mapper-placed basic supplies. Spawning moved to code rather than hand-placed items on the map. High-risk (even if only flavorfully) areas and areas far from any other kit/medical have had a wall locker added. Lockers using the same /obj for other purposes (usually blood lockers) have been swapped to use the medical/secure variant. Meant to accompany changes to basic first aid kit contents in #20187 

image

image

### CODE REVIEW

**Comment by OolongCow** [comment]
!review

**Review by hazelrat — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/crates_lockers/closets/walllocker.dm [hunk 1]
```diff
@@ -87,6 +87,9 @@
 	desc = "It's wall-mounted storage unit for first aid supplies."
 	icon_state = "medical_wall"
 
+/obj/structure/closet/walllocker/medical/fill()
+	new /obj/item/storage/firstaid/regular
```
[the remark this answers is not in the document]
Comment (DreamySkrell) [reply code/game/objects/structures/crates_lockers/closets/walllocker.dm:91]: Requesting above changes.

Base closet/locker types should be empty, and there should be named subtypes that start with items.

**Review by DreamySkrell — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/crates_lockers/closets/walllocker.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (OolongCow) [reply code/game/objects/structures/crates_lockers/closets/walllocker.dm:91]: Ah. I missed that.

[the remark this answers is not in the document]
Comment (OolongCow) [reply code/game/objects/structures/crates_lockers/closets/walllocker.dm:91]: Resolved in the new PR #20194

**Comment by OolongCow** [comment]
Closing and moving to a new PR #20194  due to a branch name change causing my commits to stop appearing.

**Pull request closed without merging**